vms/event/services.py

```diff
@@ -163,3 +166,11 @@ def remove_empty_events_for_volunteer(event_list, volunteer_id):
         if job_list:
             new_event_list.append(event)
     return new_event_list
+
+def remove_invalid_events(event_list):
+    """Removes all the events from an event list which have been occured already(expired)"""
```

Comment:
Please use 
""" 
Text 
"""

Reply:
https://github.com/systers/vms/blob/develop/vms/job/services.py#L114
I had followed the structure as used in other functions.
Making the changes might break the consistency. Should I proceed?
Thanks.

Reply:
okey - so its acceptable to have it on the same line - but since your text is long > 72 - you might want to split, and then go for
"""" 

"""

If not, this is good enough as per https://www.python.org/dev/peps/pep-0008/#id33